pumpkin/src/client/player_packet.rs

```diff
@@ -661,17 +662,22 @@ impl Player {
                     }
 
                     let block_bounding_box = BoundingBox::from_block(&world_pos);
-                    let bounding_box = entity.bounding_box.load();
-                    //TODO: Make this check for every entity in that posistion
-                    if !bounding_box.intersects(&block_bounding_box) {
+
+                    //Search in 20 block radius
+                    let mut intersects = false;
+                    for player in world.get_nearby_players(entity.pos.load(), 20).await {
```

Comment:
I would not say no, because the block should not intercept with other boundingboxes. I mean the position of the player is unimportant for the collision check. As you can see the radius is 20 blocks so the "placer" or player that places the block will automatically be in that range so players cannot place in themselfs. I hope I was able to make clear why used the block as the middle of the searching circle. But if you still want that to be changed I can do that.